proto/osmosis/superfluid/query.proto

```diff
@@ -125,12 +125,20 @@ service Query {
                                    "unpool_whitelist";
   }
 
-  rpc UserSuperfluidPositionsPerConcentratedPoolBreakdown(
-      UserSuperfluidPositionsPerConcentratedPoolBreakdownRequest)
-      returns (UserSuperfluidPositionsPerConcentratedPoolBreakdownResponse) {
+  rpc UserConcentratedSuperfluidPositionsBonded(
+      UserConcentratedSuperfluidPositionsBondedRequest)
```

Comment:
Naming nit again: what are your thoughts on matching our staked gamm share convention of reverring to "bonded/staked" shares/positions as "delegated" (and "undelegated") as we already do?

Reply:
I can name delegated and undelegating, undelegated seems like the wrong name though

Reply:
Yeah that's good, I was just saying it should be permutations of "delegate" instead of "bond"

Reply:
Done here https://github.com/osmosis-labs/osmosis/pull/5691/commits/88fb86bfa7cf2518461879a8c2092ce9a831e48f